loopy/symbolic.py

```diff
@@ -845,17 +847,22 @@ def __init__(self, swept_inames, subscript):
         self.swept_inames = swept_inames
         self.subscript = subscript
 
-    def get_begin_subscript(self):
+    def get_begin_subscript(self, kernel):
```

Comment:
Make this a function, not a method.

Reply:
Done in 1363a694. But, I would be curious to know the reason behind it.